.gitignore

```diff
@@ -5,3 +5,4 @@ bower_components/
 /public/main.css
 /pa11y_screenCapture/*
 *.env*
+.idea/
```

Comment:
could you put this in your global .gitignore - we've had problems in the past (still not totally resolved) with .gitignores in repos full of lots of text editor specific lines